sdk/push/src/main/java/com/klaviyo/push/KlaviyoPushService.kt

```diff
@@ -26,6 +39,47 @@ class KlaviyoPushService : FirebaseMessagingService() {
         fun getCurrentPushToken(): String {
             return KlaviyoPreferenceUtils.readStringPreference(PUSH_TOKEN_PREFERENCE_KEY) ?: ""
         }
+
+        /**
+         * Track an opened push event to klaviyo
+         *
+         * TODO customer and event properties should both be optional, once SDK is tracking customer state internally
+         *
+         * @param pushPayload The data attribute of the push notification payload
+         */
+        fun handlePush(
```

Comment:
note I think we're going to rename this method, but for the moment I copied what we call it in iOS. I've suggested `openedPush` or `trackPush` in the google doc

Reply:
I'm ok with this but thats probably just cause thats what I'm used to finding it as in the iOS one ya. `openedPush` is the most direct so I like that best but I think for now consistency between the two SDKs is better!

Reply:
👍  Doc I was referring to https://docs.google.com/document/d/1CYKLxGBIax61qvnRyhk2z8tVF16im84F7hVHMGtiuzE/edit#heading=h.13lek0b86grb

Reply:
Ya that lines up with what I saw in actual code too I believe